.ci/install_crio.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+get_cc_versions
+
+echo "Get CRI-O sources"
+crio_repo="github.com/kubernetes-incubator/cri-o"
+go get -d "$crio_repo" || true
+pushd "${GOPATH}/src/${crio_repo}"
+git fetch
+git checkout "${crio_version}"
+
+# Add link of go-md2man to $GOPATH/bin
+GOBIN="$GOPATH/bin"
+if [ ! -d "$GOBIN" ]
+then
+        mkdir -p "$GOBIN"
+fi
+ln -s $(command -v go-md2man) "$GOBIN"
+
+echo "Get CRI Tools"
+critools_repo="github.com/kubernetes-incubator/cri-tools"
+go get "$critools_repo" || true
+pushd "${GOPATH}/src/${critools_repo}"
+crictl_commit=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
+git checkout "${crictl_commit}"
+go install ./cmd/crictl
+sudo install "${GOPATH}/bin/crictl" /usr/bin
+popd
+
+echo "Installing CRI-O"
+make clean
+make install.tools
+make
+make test-binaries
+sudo -E PATH=$PATH sh -c "make install"
+sudo -E PATH=$PATH sh -c "make install.config"
+
+containers_config_path="/etc/containers"
+echo "Copy containers policy from CRI-O repo to $containers_config_path"
+sudo mkdir -p "$containers_config_path"
+sudo cp test/policy.json "$containers_config_path"
```

Comment:
Might be better as `sudo install --mode=$mode test/policy.json "$containers_config_path"` (where `$mode` = `0444`?)

Same comment for file `${crio_service_file}` below.